EIPS/eip-6617.md

```diff
@@ -40,32 +40,94 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 
 *Note* The following specifications use syntax from Solidity `0.8.7` (or above)
 
+```solidity
+pragma solidity ^0.8.7;
+
+/**
+    @title EIP-6617 Bit Based Permission
+    @dev See https://eips.ethereum.org/EIPS/eip-6617
+*/
+library EIP6617 {
+        
+	/**
+        @notice Check if _permission is a superset of _requiredPermission
+        @param _permission          The given permission
+        @param _requiredPermission  The required permission
+        @return                     True if the _permission is a superset of the _requiredPermission else False
+    */
+    function checkPermissions(uint256 _permission, uint256 _requiredPermission) internal pure returns (bool) {
```

Comment:
I think, we should change these three methods to

- permissionCheck
- permissionGrant
- permissionRevoke

Just like we discussed